### PR TITLE
Add SharingDomainRestrictionMode examples

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/3914

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -251,6 +251,22 @@ Set-SPOTenant -EnableAutoExpirationVersionTrim $false -MajorVersionLimit 500 -Ex
 
 This example sets Manual Version Storage Limits on all new document libraries at Tenant Level by limiting the number of major versions with no time limits.
 
+### EXAMPLE 14
+
+```powershell
+Set-SPOTenant -SharingDomainRestrictionMode "AllowList" -SharingAllowedDomainList "contoso.com"
+```
+
+This example sets that users will be able to share with external collaborators coming only from that email domain.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:260]:
```suggestion
This example enables users to share with external collaborators from that email domain only.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 2]
```diff
@@ -251,6 +251,22 @@ Set-SPOTenant -EnableAutoExpirationVersionTrim $false -MajorVersionLimit 500 -Ex
...
+
+### EXAMPLE 15
+
+```powershell
+Set-SPOTenant -SharingDomainRestrictionMode "BlockList" -SharingBlockedDomainList "contoso.com"
+```
+
+This example sets that users will be able to share with all external collaborators apart from the ones on the BlockedDomainList.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:268]:
```suggestion
This example enables users to share with all external collaborators except for those on the BlockedDomainList.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@get-itips

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Comment by yogkumgit** [comment]
@get-itips

**Comment by yogkumgit** [comment]
@tseward

**Comment by yogkumgit** [comment]
@tseward Gentle reminder.

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**